src/Kernel/BlockClosure.class.st

```diff
@@ -510,6 +512,40 @@ BlockClosure >> on: exception fork: handlerAction [
 			  nil ]
 ]
 
+{ #category : #exceptions }
+BlockClosure >> on: exception fork: handlerAction return: answerAction [
```

Comment:
This method was packaged in UnifiedFFI. I think it should be closer to no #return: version.